java/clients/src/main/java/sleeper/clients/admin/properties/AdminClientPropertiesStore.java

```diff
@@ -103,6 +103,15 @@ private List<String> listTables(InstanceProperties instanceProperties) {
                 .map(TableIdentity::getTableName).collect(Collectors.toUnmodifiableList());
     }
 
+    public List<String> listOnlineTables(String instanceId) {
+        return listOnlineTables(loadInstanceProperties(instanceId));
+    }
+
+    private List<String> listOnlineTables(InstanceProperties instanceProperties) {
+        return new DynamoDBTableIndex(instanceProperties, dynamoDB).streamOnlineTables()
+                .map(TableIdentity::getTableName).collect(Collectors.toUnmodifiableList());
+    }
```

Comment:
I don't think these need to be in this class? Note that the admin client already has a TableIndex.

We could also remove the listTables methods.